source/includes/_reference.md

```diff
@@ -2701,6 +2666,10 @@ DELETE /listen/:channel_id/:resource_type/:resource_id
 client.unwatchCorpus('555daefff80f910100d741d6')
 ```
 
+```javascript
+client.unwatchCorpus('555daefff80f910100d741d6', listener);
```

Comment:
It is the same function passed to watchCorpus. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener works that way, the function passed to addEvenListener needs to be passed to removeEventListener.
I guess the python API remove all the listeners ?